cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -163,6 +164,13 @@ func runApply(flags *applyFlags, args []string) error {
 		return errors.Wrap(err, "[upgrade/apply] FATAL")
 	}
 
+	// TODO: https://github.com/kubernetes/kubeadm/issues/2200
+	fmt.Printf("[upgrade/postupgrade] Applying label %s='' to Nodes with label %s=''\n",
```

Comment:
```suggestion
	fmt.Printf("[upgrade/postupgrade] Applying label %s='' to Nodes with label %s='' (deprecated)\n",
```
Let's make it explicit for the users that the old label is now deprecated